theme/cobalt2.json

```diff
@@ -15,8 +15,7 @@
     },
     "type": {
       // "foreground": "#ff0088",
-      "foreground": "#FF68B8",
-      "italic": true
+      "foreground": "#FF68B8"
```

Comment:
Not sure if we can do this, as this change will affect all languages not just Python.